app/components/works/work_type_modal_component.html.erb

```diff
@@ -1,50 +1,64 @@
 <script>
-  document.subtypes = <%= WorkType.to_json.html_safe %>
+  document.subtypes = <%= WorkType.to_json.html_safe %>;
+  document.moreTypes = <%= WorkType.more_types.to_json.html_safe %>
 </script>
 
-<div class="modal-header">
```

Comment:
I recommend reviewing this diff w/ whitespace turned off in GitHub PR review settings.